test/BinTestCases.test.js

```diff
@@ -0,0 +1,145 @@
+/* globals describe it before */
+"use strict";
+
+const path = require("path");
+const fs = require("fs");
+const spawn = require("child_process").spawn;
+
+function loadOptsFile(optsPath) {
+	// Options file parser from Mocha
```

Comment:
Comment can be removed as we decided to stick with Jest

Reply:
Nice catch!

Reply:
Actually, this should be kept, if you see next line